homeassistant/components/aurora/strings.json

```diff
@@ -25,5 +25,17 @@
         }
       }
     }
+  },
+  "entity": {
+    "binary_sensor": {
+      "visibility_alert": {
+        "name": "Visibility alert"
+      }
+    },
+    "sensor": {
+      "visibility": {
+        "name": "Visibility"
```

Comment:
Was the removal of the '%' intentional? I see it was retained on the 'threshold' entry above.

Reply:
What do you mean with threshold?

We don't add the unit of measurement to the name so that's what I'm removing here as well.

Reply:
[Earlier in strings.json](https://github.com/home-assistant/core/blob/caa3aef148c948215ae85eaaf113ab3c7db29880/homeassistant/components/aurora/strings.json#L24) there is a 'Threshold (%)'.

Reply:
But now I see that's probably for the options screen where you do want the units visible.

Reply:
Aah, I get what you mean, I'll fix that in another PR as that isn't an entity translation

Reply:
> But now I see that's probably for the options screen where you do want the units visible.

Yea maybe I'll change that to visibility percentage or something along those lines